src/hooks/useScrollOnRouteChange.ts

```diff
@@ -3,12 +3,24 @@ import history from 'routerHistory'
 
 const useScrollOnRouteChange = () => {
   useEffect(() => {
+    const isFirefox = navigator.userAgent.indexOf('Firefox') !== -1
+
     const unlisten = history.listen(() => {
-      window.scrollTo({
-        top: 0,
-        left: 0,
-        behavior: 'smooth',
-      })
+      if (isFirefox) {
+        setTimeout(() => {
+          window.scroll({
+            top: 0,
+            left: 0,
+            behavior: 'smooth',
+          })
+        }, 50)
+      } else {
+        window.scroll({
+          top: 0,
+          left: 0,
+          behavior: 'smooth',
+        })
+      }
```

Comment:
Lets just add the setTimeout on everything to simplify the code.

Reply:
Done